### ISSUE TITLE
attention疑问

### ISSUE THREAD

**Issue by sunpenglv** [YeexiaoZheng/Multimodal-Sentiment-Analysis#4]
attention_out = self.attention(torch.cat(
            [text_feature.unsqueeze(0), img_feature.unsqueeze(0)],
        dim=2)).squeeze()
        
请问，上面代码是在对同一batch内的不同样本进行attention操作么？

**Comment by YeexiaoZheng** [comment]
是OTEModel吗？这个model我在写的时候好像是有点问题的，但是不知道为何准确率会高

**Comment by YeexiaoZheng** [comment]
![image](https://github.com/YeexiaoZheng/Multimodal-Sentiment-Analysis/assets/97672942/bcc465cd-6341-47f0-8b5d-9788ca455ef8)

**Comment by sunpenglv** [comment]
> ![image](https://user-images.githubusercontent.com/97672942/263523098-bcc465cd-6341-47f0-8b5d-9788ca455ef8.png)

请问这个图有链接么？

**Comment by sunpenglv** [comment]
> 是OTEModel吗？这个model我在写的时候好像是有点问题的，但是不知道为何准确率会高

是OTEModel模型。

**Comment by YeexiaoZheng** [comment]
这是我的实验报告，具体attention的方式还是看看别人的论文写写吧，我的有点排脑瓜子想的成分在
